README.md

```diff
@@ -77,21 +77,30 @@ int main() {
         "elbow_angle");
     model.addComponent(reporter);
 
+    // Add display geometry.
+    Ellipsoid* bodyGeometry = new Ellipsoid(0.1, 0.5, 0.1);
+    bodyGeometry->setColor(Gray);
+    // Attach an ellipsoid to a frame located at the center of each body.
+    PhysicalOffsetFrame* humerusCenter = new PhysicalOffsetFrame(
+        "humerusCenter", "humerus", Transform(Vec3(0, 0.5, 0)));
+    humerus->addComponent(humerusCenter);
+    humerusCenter->attachGeometry(bodyGeometry->clone());
```

Comment:
Yes, I originally had `clone()` on only one of the calls to `attachGeometry()` but changed it to preserve symmetry, which I thought would be appropriate for this didactic example. I've adopted @chrisdembia's (i) and @aseth1's slight preference.